src/Sylius/Bundle/ShopBundle/Resources/translations/flashes.ar.yml

```diff
@@ -0,0 +1,4 @@
+sylius:
+    cart:
+        product:
+            not_available: 'بعض المنتجات غير متوفرة وتم إزالتها من السلة.'
```

Comment:
Please add a trailing newline at the end of this and other `flash.*.yml` files.  
Also, rename their extensions from `.yml` to `.yaml` as we use `.yaml` across the project.